docs/Support.md

```diff
@@ -0,0 +1,38 @@
+# Support
+
+## I need help right now
+
+1. Don't panic
+2. Page MoFo on-call support with a clear description of the problem and a method to contact you (heading to #fundraising on irc.mozilla.org is a good idea)
```

Comment:
I might be missing it, but do we want any info on what mofo on-call is and how to call it?